aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -3010,8 +3024,12 @@ pub(crate) fn should_create_account_resource(
         && txn_data.replay_protector == ReplayProtector::SequenceNumber(0)
     {
         let account_tag = AccountResource::struct_tag();
+        assert!(account_tag.address.is_special());
```

Comment:
is this an easily checkable / local invariant from the code?

Reply:
I think so, account resource is always at 0x1, I also added a test to check for that with a comment that loading asserts this.